README.md

```diff
@@ -35,6 +35,10 @@ This project has adopted the [Microsoft Open Source Code of Conduct](https://ope
 
 You must sign a [Contributor License Agreement](https://cla.microsoft.com/) before submitting your pull request. To complete the Contributor License Agreement (CLA), you will need to submit a request via the [form](https://cla.microsoft.com/) and then electronically sign the CLA when you receive the email containing the link to the document. You need to sign the CLA only once to cover submission to any Microsoft OSS project. 
 
+### 2.3 Code Formatting
+
+All Objective-C files follow LLVM coding style (with a few exceptions) and formatted accordingly. If you use Xcode for your contribution, use [.clang_format](https://github.com/Microsoft/AppCenter-SDK-Apple/blob/develop/.clang-format) file to make sure files have the correct code format before submitting PRs.
```

Comment:
`and formatted accordingly` -> `and are formatted accordingly`
`use [.clang_format]` -> `use our [.clang_format]`

Reply:
Maybe less verbose like this:
```All Objective-C files are formatted according to LLVM coding style (with a few exceptions). If you use Xcode, please use our [.clang_format](https://github.com/Microsoft/AppCenter-SDK-Apple/blob/develop/.clang-format) before submitting PRs.```

Reply:
@dhei I wanted to emphasize to run code formatter before PR. Is it too verbose?

Reply:
@jaelim-ms Either one is clear enough. I personally prefer the concise version, but if your intention is emphasizing code formatting, that is perfectly fine.